README.md

```diff
@@ -10,44 +10,76 @@ required to deploy to different networks.
 Install ZeppelinOS, Ganache, and Truffle
 
 ```
-npm install -g truffle@5.0.2 ganache-cli@6.3.0 zos@2.2.0
+npm install -g truffle@5.0.2 ganache-cli@6.3.0 zos
 ```
 
 ## Installation
 
 First ensure you are in a new and empty directory.
 
-Run the `unbox` command.
+Run the `unpack` command with `zepkit` to create a clean project. 
```

Comment:
```suggestion
Run the `unpack` command with `zepkit` to create a starter project. 
```
Let's start to use starter term because kits will be renamed to OpenZeppelin Starter Kit.